app/src/main/java/org/simple/clinic/recentpatientsview/RecentPatientsViewController.kt

```diff
@@ -64,7 +67,12 @@ class RecentPatientsViewController @Inject constructor(
         .refCount()
 
     val updateRecentPatients = recentPatientsStream
-        .map { it.map(::recentPatientItem) }
+        .map {
+          it.map { recentPatient ->
+            val today = Instant.now(userClock).toLocalDateAtZone(userClock.zone)
```

Comment:
This will still end up creating a new instance for every patient item. We should leave it inside the Rx `.map` operator block, but outside the list `.map` block.

Reply:
```kotlin
.map { 
  val today = LocalDate.now(userClock)
  it.map { recentPatient -> recentPatientItem(recentPatient, today) } 
}
```

Reply:
Ah right, sorry 😓

Reply:
Also, we don't need to create an `Instant` to get a `LocalDate`. We can just create a `LocalDate` directly using `LocalDate.now(Clock)`.

Reply:
Yes, forgot about it. Changing it to `LocalDate`